### ISSUE TITLE
Improve handling of secured connections in Camunda Modeler

### ISSUE THREAD

**Issue by nikku** [camunda/camunda-modeler#3563]
### Problem you would like to solve

We've seen many examples where our existing deployment experience to self-managed hurts our users:

* They need to feed `http` vs. `https` as a URL to denote insecure vs. secure connection. This is different from all other clients where rather "secured" is set explicitly via a flag
* The fact that `http*` is used as a protocol deludes customers in thinking this is a text-based protocol. While tunneling happens via `HTTP/2` it is actually a binary protocol that must be handled accordingly, i.e. by putting proxies in binary proxy mode (https://docs.camunda.io/docs/next/self-managed/platform-deployment/troubleshooting/#zeebe-ingress-grpc). 
* The fact that customers enter a URL is miss-leading, too (and different from any other client). The thing to connect to is an `ADDRESS` (`HOST_NAME:PORT`) and path parameters are not supported.

### Proposed solution

* Get rid of the URL in favor of entering `ADDRESS` (`HOST_NAME(:PORT)*`) and an explicit `SECURED` flag (i.e. a checkbox)

### Alternatives considered

* Get rid of `http` as a prefix in favor of the actual protocol `grpc`

### Additional context

Potential places where this caused confusion:

* SUPPORT-16716

**Comment by nikku** [comment]
CC @YanaSegal @christian-konrad

**Comment by barmac** [comment]
Note however that for the simplest local test you want to connect insecurely: `docker run -p 26500:26500 zeebe:latest`.

**Comment by nikku** [comment]
> https://github.com/camunda/camunda-modeler/issues/3563#issuecomment-1517528753

Yes. But even then we'd need to specify `http://localhost:26500` as a URL, right? So how does that relate to the described problem and solution approaches?

**Comment by barmac** [comment]
Hmm I did not notice that you mentioned the checkbox for secure/insecure connection 🤦

**Comment by nikku** [comment]
Moving to backlog as this is not our immediate priority.

**Comment by nikku** [comment]
https://github.com/camunda/camunda-modeler/issues/5103 proposes the alternative suggested: Simply use the `grpc(s)` or `http(s)` prefix to tell apart `GRPC over HTTP/2` from `REST over HTTP` connections.
